roles/confluent.test/molecule/custom-user-plaintext-rhel/molecule.yml

```diff
@@ -89,12 +95,12 @@ provisioner:
         zookeeper_user: cp-test
         zookeeper_group: cp-test-group
         zookeeper:
-          log_path: /kafka/logs/
+          log_path: /kafka/logs
```

Comment:
how serious do we want to be about backwards compatibility here?

Reply:
could use a filter to remove final / if there is one just to be safe: https://firxworx.com/blog/it-devops/sysadmin/using-ansibles-regex_replace-filter-to-strip-leading-and-trailing-slashes-from-strings/